### PR TITLE
Update the tutorial for elasticdl local run.

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

File: docs/tutorials/elasticdl_local.md [hunk 1]
```diff
@@ -23,10 +25,9 @@ we use a model predefined in model zoo directory.
 ### Install ElasticDL
 
 ```bash
+pip install elasticdl_client
 git clone https://github.com/sql-machine-learning/elasticdl.git
 cd elasticdl
-pip install -r elasticdl/requirements.txt
-python setup.py install
 ```
```
Comment (QiJune) [inline docs/tutorials/elasticdl_local.md:35]: ### Install ElasticDL Client

```bash
pip install elasticdl_client
```

Clone elasticdl repo for model zoo and some scripts.

```bash
git clone https://github.com/sql-machine-learning/elasticdl.git
```

### Setup Kubernetes related environment

```bash
minikube start --vm-driver=hyperkit --cpus 2 --memory 6144 --disk-size=50gb --mount=true --mount-string="$DATA_PATH:/data"
cd elasticdl
kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
eval $(minikube docker-env)
```

Mount the host path $DATA_PATH to /data of minikube

### Prepare dataset

```
docker pull elasticdl/elasticdl:dev
cd elasticdl
docker run --rm -it \
  -v $HOME/.keras/datasets:/root/.keras/datasets \
  -v $PWD:/work \
  -w /work elasticdl/elasticdl:dev \
  bash -c "scripts/gen_dataset.sh $DATA_PATH"

```


### Submit job

--volume="/data,mount_path=/data"

mount /data of minikube to /data of master pod

Comment (QiJune) [reply docs/tutorials/elasticdl_local.md:35]: Please refer to https://stackoverflow.com/questions/48534980/mount-local-directory-into-pod-in-minikube/48535001

File: docs/tutorials/elasticdl_local.md [hunk 2]
```diff
@@ -6,11 +6,13 @@ the working process of ElasticDL.
 
 ## Environment preparation
 
-Here we should install Minikube first. Please refer to the official
+1. Install Minikube first. Please refer to the official
 [installation guide](https://kubernetes.io/docs/tasks/tools/install-minikube/).
```
Comment (QiJune) [inline docs/tutorials/elasticdl_local.md:10]: In this tutorial, minikube version is `v1.11.0`

File: docs/tutorials/elasticdl_local.md (hunk 1 above)
Comment (brightcoder01) [reply docs/tutorials/elasticdl_local.md:35]: Great comments. updated.

File: docs/tutorials/elasticdl_local.md (hunk 2 above)
Comment (brightcoder01) [reply docs/tutorials/elasticdl_local.md:10]: Good catch, updated.

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**